src/data-loader.ts

```
@@ -794,6 +795,55 @@ export async function calculateCostForEntry(
 		return 0;
 	}
 
+	if (mode === 'statusline') {
+		// Statusline mode: saved statusline costs → pre-calculated costUSD → token-based calculation
+		if (data.sessionId != null) {
```

ESLint

[error] Unsafe member access .sessionId on an `error` typed value.

(ts/no-unsafe-member-access)